analysis_schema/data_classes.py

```diff
@@ -171,10 +232,26 @@ class PhasePlot(ytBaseModel):
     Comments: Optional[str]
     _yt_operation: str = "PhasePlot"
 
+    def _run(self):
+        super_list = []
+        if self.ds is None:
+            # self.ds = list(DatasetFixture._instantiated_datasets.values())[0]
+            for instantiated_keys in list(
+                dataset_fixture._instantiated_datasets.keys()
+            ):
+                self.ds = dataset_fixture._instantiated_datasets[instantiated_keys]
+                super_list.append(super()._run())
+                # put each 'self' into the output
+                # when calling `._run()` there is no plotting
+                # attribute, so it is not added to the output list
```

Comment:
I think you'd want the same `if self.ds is not None` as the other plots? Which might make this identical to the other plot _runs?

Reply:
Yes, I think we want the same behavior for the `_run` function for both `SlicePlot` and `ProjectionPlot`, but I don't think this will quite work for `PhasePlot`, because if I'm remembering correctly the `ds` functions differently in that there.

Reply:
ah, right. I'll take a look, but let's not worry about this for the current PR.

Reply:
(you can resolve this convo too!)